packages/components/src/Disclosure/Disclosure.css

```diff
@@ -0,0 +1,64 @@
+/*
+Re: (.arrowIconWrapper > svg > path)
+This ensure we are spinning just the svg graphic and not the
+conter, which result in an odd square rendering glitch when
+displaying the background color together.
+*/
+.arrowIconWrapper > svg > path {
+  transform-origin: 50% 50% 0;
+  transition: transform var(--timing-base);
+}
+
+.flippedVertical > svg > path {
+  transform: rotate(180deg);
+}
+
+.summary {
+  display: flex;
+  border: none;
+  list-style: none;
+  cursor: pointer;
+  outline-color: none;
+}
+
+.summary::-webkit-details-marker {
+  display: none;
+}
+
+.summary:hover {
+  background: var(--color-yellow--lightest);
+}
+
+.summary:focus,
+.summary:active {
+  border: none;
+  outline-color: var(--color-yellow);
```

Comment:
~can we use `var(--color-focus)` for this?~

Reply:
We use `var(--shadow-focus)` rather than `outline` to convey :focus state where possible:
![image](https://user-images.githubusercontent.com/39704901/114586589-13451080-9c42-11eb-89a1-724022b27c76.png)
![image](https://user-images.githubusercontent.com/39704901/114586661-26f07700-9c42-11eb-86fa-c26c273f2d12.png)